README.md

```diff
@@ -81,6 +81,10 @@ If you have an issue, please create one. But, before:
 - try to run it with `--debug` flag and see the output
 - try to make create repro example
 
+## Building
+
+Run `tsc --build tsconfig.build.json` to transpile this project.
```

Comment:
… just because I had to look and experiment for a while to figure out how to compile this project.